src/index.d.ts

```diff
@@ -469,6 +469,12 @@ declare namespace admin.messaging {
     launchImage?: string;
   };
 
+  type CriticalSound = {
+    critical?: boolean;
+    name?: string;
```

Comment:
is it optional? What happens if you don't provide a name?

Reply:
APNs docs are not clear on that point: https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification#2990112

I believe it will default to system sound if none specified.

Reply:
Will make sure this gets documented in the published docs.